ross/tests/test_disk_element.py

```diff
@@ -11,16 +11,24 @@ def disk():
 
 
 def test_mass_matrix_disk(disk):
-    Md1 = np.array([[ 32.58973,   0.     ,   0.     ,   0.     ],
-                    [  0.     ,  32.58973,   0.     ,   0.     ],
-                    [  0.     ,   0.     ,   0.17809,   0.     ],
-                    [  0.     ,   0.     ,   0.     ,   0.17809]])
+    Md1 = np.array(
```

Comment:
Was this the result of running black on the file?
For these cases, I believe we can use the `# fmt: off` and `# fmt: on`. This will make black skip some blocks of code. 
I think this could be useful for some formulas and some matrices where the 'human format' is better than the black output to compare with textbooks when trying to find errors. 
Anyway, I don't think this is a problem now and we can modify these matrices latter as we wish. 
@flaviorangel I think we should add to the documentation a part related to the use of black, and we should highlight that we can have this exception where we can use `# fmt: off` for some equations and matrices.